__tests__/app.test.js

```diff
@@ -501,3 +501,28 @@ describe('api/comments/:comment_id', () => {
     });
   });
 });
+
+describe('GET /api', () => {
+  test('status:200 and responds with JSON which has the correct keys', () => {
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body).toBe('object');
+        const validAPIs = [
+          'GET /api',
+          'GET /api/topics',
+          'GET /api/articles',
+          'GET /api/articles/:article_id',
+          'PATCH /api/articles/:article_id',
+          'GET /api/users',
+          'GET /api/articles/:article_id/comments',
+          'POST /api/articles/:article_id/comments',
+          'DELETE /api/comments/:comment_id',
+        ];
```

Comment:
lovely test for this but remember that you'll need to update this whenever you add to you .json file